app/src/main/java/org/simple/clinic/overdue/AppointmentRepository.kt

```diff
@@ -32,7 +32,7 @@ class AppointmentRepository @Inject constructor(
     private val appointmentConfigProvider: Single<AppointmentConfig>
 ) : SynceableRepository<Appointment, AppointmentPayload> {
 
-  fun schedule(patientUuid: UUID, appointmentDate: LocalDate): Single<Appointment> {
+  fun schedule(patientUuid: UUID, appointmentDate: LocalDate, isDefaulter: Boolean?): Single<Appointment> {
```

Comment:
I know that this will change, but I'd like to give feedback on this design.

The `isDefaulter` flag might be `nullable` when defined in the entity, but that is because we have a limitation when migrating. However methods we expose from the API should behave as if the flag was not nullable, so this should be a `Boolean`.